README.rdoc

```diff
@@ -3,7 +3,7 @@
 Portrait is a web service which converts a website into a image. The actual hard work is handled by Google's {Puppeteer}[https://github.com/GoogleChrome/puppeteer].  Portrait is primarily designed to be used as a service, but does have a minimal admin interface.
 
 == Dependencies
-You'll need to be able to run Puppeteer via Node.js. You'll also need to be able to run a Rails stack. Portrait is built on Ruby 2.3.3 and Rails 5.2
+You'll need to be able to run Puppeteer via Node.js (v7.6.0+). You'll also need to be able to run a Rails stack. Portrait is built on Ruby 2.3.3 and Rails 5.2
```

Comment:
We are using `async()` in our `generate_pdf.js` script, which is apparently only supported in Node.js v7.6.0 or greater